nuxeo-data-table/iron-data-table.html

```diff
@@ -365,7 +348,7 @@ <h2>[[i18n('command.add')]]</h2>
        */
       class DataTable
         extends Polymer.mixinBehaviors([Polymer.IronResizableBehavior, Polymer.IronFormElementBehavior,
-          Polymer.IronValidatableBehavior, Nuxeo.PageProviderDisplayBehavior, Nuxeo.I18nBehavior,
+          Polymer.IronValidatableBehavior, Nuxeo.PageProviderDisplayBehavior,
```

Comment:
We should remove the `* @appliesMixin Nuxeo.I18nBehavior` from the documentation of this element as well. (line 343)

Reply:
:+1: